.storybook/pages/CadStep1/CadStep1.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+
+import {
+  Button,
+  ButtonGroup,
+  Heading,
+  HorizontalStepper,
+  Icon,
+  Layout,
+  LayoutContainer,
+  LayoutSection,
+  PageHeader,
+  Panel,
+  Text,
+} from '../../../src';
+import TextField from '../../../src/upcoming-components/TextField';
+
+import '../../../src/components/Utilities/Spacing.css';
+import '../../../src/components/Utilities/TypographyUsage.css';
+
+export const CadStep1 = () => (
+  <>
+    <Panel className="u-margin-bottom-xl" variant="squared">
+      <LayoutContainer>
+        <PageHeader
+          className="u-margin-bottom-none"
+          description={
+            <HorizontalStepper
+              activeIndex={0}
+              steps={[
+                'Add classroom details',
+                'Add projects',
+                'Create course plan',
+              ]}
+            />
+          }
+          headingSize="h5"
+          right={
+            <ButtonGroup>
+              <Button>Return to Course</Button>
+              <Button variant="primary">
+                Next <Icon name="arrow-narrow-right" purpose="decorative" />
+              </Button>
+            </ButtonGroup>
+          }
+          title="Course Planner: History 6"
+        />
+      </LayoutContainer>
+    </Panel>
+    <LayoutContainer>
+      <Layout variant="50-50">
+        <LayoutSection className="u-margin-bottom-xl" region="main">
+          <Heading className="u-margin-bottom-xl" size="h1">
+            Welcome, Amy. Chart your course for the year here.
+          </Heading>
+          <Text className="u-margin-bottom-xl" size="lg">
+            The course planner can help you set off on the right track, and
+            navigate whatever comes your way.
+          </Text>
+          <svg fill="none" viewBox="0 0 505 365" width="92%">
```

Comment:
Wondering if we create a `Picture` component that has some logic baked in for teams to use to render different sizes/orientations of images on different screen sizes. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture

Reply:
Created a ticket for making `picture` component, and is a little WIP https://app.shortcut.com/czi-edu/story/198344/create-responsive-picture-component, but likely sidelining for higher pri tix

Reply:
Mocks changed so won't be needing responsive images